test/app.test.js

```diff
@@ -2,7 +2,12 @@ import React from 'react'
 import { render } from '@testing-library/react'
 
 import App from '../src/components/App'
+import Signup from '../src/components/Authentications/RegisterUser'
 
 test('Initial render, Welcome to BareFoot Nomad frontEnd Tests', () => {
   render(<App />)
 })
+test('It should test all Sign up lines', () => {
+  render(<Signup />)
+})
+
```

Comment:
add a signup test file separately and add signup tests.
test the inputs individually as well.